src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -72,7 +72,18 @@ export default class DiscussionArea {
   addSeekingListener() {
     this.#eventController.addEventListener((timestampMs) => {
       this.seek(timestampMs);
-    }, 'seek');
+    }, 'seek', 'seekAll');
+  }
+
+  /**
+   * Seeks the transcript, discussion, and video to `timestampMs`.
+   *
+   * <p>This is private and should be added as an event listener to every root
+   * discussion header's onclick event.
+   */
+  onTimestampClicked(timestampMs) {
```

Comment:
what does onTimestampClicked? Which timestamp was clicked?

Reply:
renamed to `onCommentHeaderClicked`